### ISSUE TITLE
Include support for /dev/null (linux) and \\.\NUL (windows)

### ISSUE THREAD

**Issue by dutchgecko** [na-ka-na/ExcelCompare#29]
Currently, using `git diff HEAD~1` on a newly committed excel file produces the error `File: /dev/null is not a file.` on linux. This is because git requests the diff command to compare between the new file and /dev/null (the "empty" file). Ideally ExcelCompare would treat /dev/null as a real, empty excel file, and show all additions made.

Relevant git configuration:

~/.gitconfig

```
[diff "xls"]
    binary = true
    command = ~/bin/ExcelCompare/bin/excel_cmp
```

~/.config/git/attributes

```
*.xls diff=xls
```

**Comment by na-ka-na** [comment]
Done, added support for /dev/null and \\.\NUL. Haven't tested on windows though since I don't have a windows machine anymore.

**Comment by na-ka-na** [comment]
Changes are in just released 0.6.0.
